content/chronograf/v1.8/administration/config-options.md

```diff
@@ -280,6 +280,14 @@ Environment variable: `$HOST_PAGE_DISABLED=true`
 
 ### General authentication options
 
+#### `--inactivity-duration=`
```

Comment:
We may want to move this after `--auth-duration`?

Reply:
Alphabetized. ✅